test/verify/check-active-pages

```diff
@@ -33,7 +33,8 @@ class TestActivePages(MachineCase):
         if m.image in [ "rhel-8-0-distropkg" ]:
             n_extra_preloaded = 0
         else:
-            n_extra_preloaded = 1
```

Comment:
rhel-8-0-distropkg is no more; however, this now applies to rhel-8-1-distropkg as well, so I believe this test needs adjustment.

Reply:
(Why wasn't it adjusted when we moved from 8-0-distropkg to 8-1-distropkg?  This PR doesn't introduce the conditional, it just adjusts the expectations in the two cases.)

Reply:
It was adjusted. This is apparently a curious UI bug in github -- in the diff it  *still* shows the condition as `if m.image in [ "rhel-8-0-distropkg" ]:`, but in master it's really 8-1. So nevermind. :-)